build_runner/test/server/hot_reload_client_test.dart

```diff
@@ -0,0 +1,85 @@
+// Copyright (c) 2017, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+@TestOn('browser')
```

Comment:
Do I need to add any new things to .travis.yml for this test to be run?

Reply:
Yes - we can talk offline :)